install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -30,6 +31,7 @@
 {{- $ipam := (coalesce .Values.ipam $defaultIPAM) -}}
 {{- $bpfCtTcpMax := (coalesce .Values.global.bpf.ctTcpMax $defaultBpfCtTcpMax) -}}
 {{- $bpfCtAnyMax := (coalesce .Values.global.bpf.ctAnyMax $defaultBpfCtAnyMax) -}}
+{{- $crdWaitTimeout := (coalesce .Values.crdWaitTimeout $defaultCrdWaitTimeout) -}}
```

Comment:
I believe he coalescence is because these options were used before `1.8` so this line should be removed